app/src/main/res/values/strings.xml

```diff
@@ -1,16 +1,10 @@
-
 <resources>
     <string name="app_name">OneList</string>
+    <string name="item_list_header">Items</string>
     <string name="action_settings">Settings</string>
-    <!-- Strings used for fragments for navigation -->
```

Comment:
Not using fragments anymore so removing these.

Reply:
Yeah.  
Smaller codebase == easier to maintain == less technical debt.